CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.3 (2021-07-28)
+
+- Fixed the Angular tree component to fire the `stateChange` event once on click. [#42](https://github.com/blackbaud/skyux-angular-tree-component/pull/42)
```

Comment:
Is "Angular tree component" the right component name? Is this really the tree node component? (And if so, does it really make a difference?)

Reply:
Well, the fixes I made were to the angular tree component. Its logic needed fixed to make the `stateChange` fire correctly.

Reply:
OK. Sorry. Didn't mean to hang the up. Looking at the files at https://github.com/blackbaud/skyux-angular-tree-component/pull/42/files, it looked like it "node" was part of the component name. ... We should probably update the date since we didn't get this out today. Sorry about that.